modin/pandas/test/test_io.py

```diff
@@ -725,23 +725,26 @@ def test_read_csv_quoting(
         )
 
     # Error Handling parameters tests
-    @pytest.mark.xfail(
+    @pytest.mark.skip(
```

Comment:
Why skip?

Reply:
These tests were marked as skipped because bad lines handling doesn't supported in the Modin `read_csv`. Previously tests were passing but it wasn't guaranteed and now we can manually skip only failed tests but, i think, it is easier to skip full tests block.